WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/details/BackupDownloadDetailsFragment.kt

```diff
@@ -30,24 +39,39 @@ class BackupDownloadDetailsFragment : Fragment() {
         val nonNullActivity = requireActivity()
         (nonNullActivity.application as WordPress).component().inject(this)
 
+        initRecyclerView()
+        initViewModel()
+    }
+
+    private fun initRecyclerView() {
+        recycler_view.layoutManager = LinearLayoutManager(context, RecyclerView.VERTICAL, false)
+        initAdapter()
+    }
+
+    private fun initAdapter() {
+        recycler_view.adapter = BackupDownloadDetailsAdapter(uiHelpers)
+    }
+
+    private fun initViewModel() {
         parentViewModel = ViewModelProviders.of(requireActivity(), viewModelFactory)
```

Comment:
Question (❓ ): How would the `parentViewModel` be used later on? Do we need it? I know this was a change from a previous PR, but since I am not seeing it being used yet, I thought I might ask now.

Reply:
@ParaskP7 - this will get flushed out as I move further along in the development. Thanks.